Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml

```diff
@@ -9,9 +9,9 @@
     <!-- XHED: empty recycler bin label -->
     <string name="recycler_bin_empty_label">"Ihr Papierkorb ist leer"</string>
     <!-- XHED: empty recycler bin description -->
-    <string name="recycler_bin_empty_description">"Wenn Sie Zertifikate entfernen, werden sie in den Papierkorb verschoben und hier angezeigt."</string>
+    <string name="recycler_bin_empty_description">"Wenn Sie Zertifikate löschen, werden sie in den Papierkorb verschoben und hier angezeigt."</string>
     <!-- XHED: recycler bin remove all menu item -->
-    <string name="recycler_bin_remove_all">"Alle entfernen"</string>
+    <string name="recycler_bin_remove_all">"Alle löschen"</string>
```

Comment:
which terms are the right ones

```xml
<<<<<<< CV113-patch-1
    <string name="recycler_bin_empty_description">"Wenn Sie Zertifikate löschen, werden sie in den Papierkorb verschoben und hier angezeigt."</string>
    <!-- XHED: recycler bin remove all menu item -->
    <string name="recycler_bin_remove_all">"Alle löschen"</string>
=======
    <string name="recycler_bin_empty_description">"Wenn Sie Zertifikate und Tests entfernen, werden sie in den Papierkorb verschoben und hier angezeigt."</string>
    <!-- XHED: recycler bin remove all menu items -->
    <string name="recycler_bin_remove_all">"Alle endgültig löschen"</string>
>>>>>>> release/2.15.x
```

Reply:
@CV113 FYI

Reply:
I don't understand what you mean!?

Reply:
Yes, because we have to replace everywhere the word ENTFERNEN with the word LÖSCHEN

Reply:
Ok, I fixed it please look at https://github.com/corona-warn-app/cwa-app-android/pull/4363/commits/700578ba01be69cd927c19ee6cf5173f92e1cbd7